client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue

```diff
@@ -400,21 +453,27 @@ export default {
         params.assigneeId = this.actions.assignSegment.selected.id
       }
 
+      if (this.assignPlaceCheckedAndSelected) {
+        params.placeId = this.actions.assignPlace.selected.id
+      }
+
       dpRpc('segment.bulk.edit', params)
         .then(checkResponse)
         .then((response) => {
           const rpcResult = this.getRpcResult(response)
 
+          this.actions.addRecommendations.success = rpcResult
```

Comment:
I'm wondering if we shouldn't create an array of actions, then for each of them set `success` to `rpcResult`. The current solution seems very verbose.

Reply:
good idea. Done here https://github.com/demos-europe/demosplan-core/pull/695/commits/e03ef8bce2ee783f7233418545ea0d7161c29c56